gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/javaversions/BaselineJavaVersionsExtension.java

```diff
@@ -31,20 +31,22 @@
  */
 public class BaselineJavaVersionsExtension {
     private final Property<JavaLanguageVersion> libraryTarget;
```

Comment:
I'm choosing to literally not support the --enable-preview functionality in this top level `javaVersions {}` extension, because I think the fact that the generate bytecode cannot be run by newer versions of java is such a massive footgun that I don't want folks to ever publish jars with this.

if people _really_ need to break glass, they can still conceivably use the subproject-level `javaVersion {}` extensions